### PR TITLE
[good to go] conveyor sorters, conveyor switch tool interacts, also makes mechs not explode on sale

### PR DESCRIPTION
**Pull request by Hatterhat**
## About The Pull Request
conveyor sorters basically taken wholesale from Skyrat-SS13/Skyrat-tg#8790 and subsequent fixes.
tl;dr: Thing Placer which places Thing that allows you to add items and autoshift them to a given direction

also adds:
tgstation/tgstation#54022
- wrench to inverse the position of a conveyor switch
- **screwdriver to lock a conveyor in one direction holy shit finally**

also also makes mechs not explode when sold maybe?
## Why It's Good For The Game
conveyor sorting funny
conveyor switch tool acts Long Overdue
mechs griefing robotics when sold unintended
## Changelog
:cl:
add: Conveyor sorters! Place them with a sorter lister (equipment tab for cargo lathe), scan items with the lister, apply to marks and tap with an empty hand to specify directions. (It makes sense in context.)
qol: Conveyor switches now have the ability to be inverted with a wrench and locked into one-way settings via screwdriver. (You care about the latter much more than the former.)
fix: Exosuits no longer explode when sold. (They were never meant to explode that hard to begin with, apparently.)
/:cl:

### CODE REVIEW

**Comment by Hatterhat** [comment]
@SandPoot that was a very quick thumbs up

**Comment by SandPoot** [comment]
Speed

**Comment by Hatterhat** [comment]
my brother in christ why are the run linters already failing

**Comment by SandPoot** [comment]
image
it unhappy about

**Comment by Hatterhat** [comment]
i blame poor ui design for making it mildly inconvenient to check where the actual problem is listed. who puts the "HEY WE HAVE A PROBLEM" indicator on the Thingy Runner and not the actual part where it gives the problem.
image

**Comment by SandPoot** [comment]
The worst problem is that, there can be either problems on run linters, annotate lints or both.

**Comment by Hatterhat** [comment]
watch me make this PR get incredibly out of scope

**Comment by Hatterhat** [comment]
upon further reflection this is probably fine to merge
